### ISSUE TITLE
Dependency Dashboard

### ISSUE THREAD

**Issue by locmai** [locmai/yuta#110]
This issue lists Renovate updates and detected dependencies. Read the [Dependency Dashboard](https://docs.renovatebot.com/key-concepts/dashboard/) docs to learn more.

## Edited/Blocked

These updates have been manually edited so Renovate will no longer make changes. To discard all commits and start over, click on a checkbox.

 - [ ] fix(deps): update google.golang.org/genproto digest to 83ca9fa
 - [ ] chore(deps): update all non-major dependencies (`aiormq`, `boto3`, `botocore`, `github.com/sirupsen/logrus`)

## Open

These updates have all been created already. Click a checkbox below to force a retry/rebase of any.

 - [ ] [chore(deps): update dependency locket to v1](../pull/107)
 - [ ] [fix(deps): update module gopkg.in/yaml.v2 to v3](../pull/108)
 - [ ] [fix(deps): update module k8s.io/client-go to v12](../pull/109)
 - [ ] **Click on this checkbox to rebase all open PRs at once**

## Detected dependencies

<details><summary>dockerfile</summary>
<blockquote>

<details><summary>Dockerfile</summary>

 - `alpine 3.16.1`

</details>

</blockquote>
</details>

<details><summary>github-actions</summary>
<blockquote>

<details><summary>.github/workflows/codeql-analysis.yml</summary>

 - `actions/checkout v3`
 - `github/codeql-action v2`
 - `github/codeql-action v2`
 - `github/codeql-action v2`

</details>

</blockquote>
</details>

<details><summary>gomod</summary>
<blockquote>

<details><summary>go.mod</summary>

 - `cloud.google.com/go/dialogflow v1.12.0`
 - `github.com/golang/protobuf v1.5.2`
 - `github.com/gorilla/mux v1.8.0`
 - `github.com/nats-io/nats.go v1.16.0`
 - `github.com/prometheus/alertmanager v0.24.0`
 - `github.com/prometheus/client_golang v1.12.2`
 - `github.com/sirupsen/logrus v1.8.1`
 - `github.com/uber/jaeger-client-go v2.30.0+incompatible`
 - `google.golang.org/genproto v0.0.0-20220713161829-9c7dac0a6568@9c7dac0a6568`
 - `gopkg.in/yaml.v2 v2.4.0`
 - `k8s.io/apimachinery v0.24.3`
 - `k8s.io/client-go v0.24.0-alpha.4`
 - `maunium.net/go/mautrix v0.11.0`
 - `github.com/googleapis/gnostic v0.6.9`

</details>

</blockquote>
</details>

<details><summary>pip_requirements</summary>
<blockquote>

<details><summary>requirements.txt</summary>

 - `absl-py ==1.2.0`
 - `aio-pika ==8.0.3`
 - `aiofiles ==0.8.0`
 - `aiohttp ==3.8.1`
 - `aiormq ==6.3.4`
 - `APScheduler ==3.9.1`
 - `astunparse ==1.6.3`
 - `async-generator ==1.10`
 - `async-timeout ==4.0.2`
 - `attrs ==21.4.0`
 - `bidict ==0.22.0`
 - `boto3 ==1.24.32`
 - `botocore ==1.27.32`
 - `CacheControl ==0.12.11`
 - `cachetools ==5.2.0`
 - `certifi ==2022.6.15`
 - `cffi ==1.15.1`
 - `chardet ==5.0.0`
 - `charset-normalizer ==2.1.0`
 - `clang ==14.0`
 - `cloudpickle ==2.1.0`
 - `colorclass ==2.2.2`
 - `coloredlogs ==15.0.1`
 - `colorhash ==1.0.4`
 - `cryptography ==37.0.4`
 - `cycler ==0.11.0`
 - `dask ==2022.7.0`
 - `decorator ==5.1.1`
 - `dm-tree ==0.1.7`
 - `dnspython ==2.2.1`
 - `docopt ==0.6.2`
 - `fbmessenger ==6.0.0`
 - `fire ==0.4.0`
 - `flatbuffers ==2.0`
 - `fsspec ==2022.5.0`
 - `future ==0.18.2`
 - `gast ==0.5.3`
 - `google-auth ==2.9.1`
 - `google-auth-oauthlib ==0.5.2`
 - `google-pasta ==0.2.0`
 - `greenlet ==1.1.2`
 - `grpcio ==1.47.0`
 - `h5py ==3.7.0`
 - `httptools ==0.4.0`
 - `humanfriendly ==10.0`
 - `idna ==3.3`
 - `importlib-metadata ==4.12.0`
 - `jmespath ==1.0.1`
 - `joblib ==1.1.0`
 - `jsonpickle ==2.2.0`
 - `jsonschema ==4.7.2`
 - `kafka-python ==2.0.2`
 - `keras ==2.9.0`
 - `Keras-Preprocessing ==1.1.2`
 - `kiwisolver ==1.4.4`
 - `locket ==0.2.1`
 - `Markdown ==3.4.1`
 - `matplotlib ==3.5.2`
 - `mattermostwrapper ==2.2`
 - `msgpack ==1.0.4`
 - `multidict ==6.0.2`
 - `networkx ==2.8.5`
 - `numpy ==1.23.1`
 - `oauthlib ==3.2.0`
 - `opt-einsum ==3.3.0`
 - `packaging ==21.3`
 - `pamqp ==3.2.0`
 - `partd ==1.2.0`
 - `Pillow ==9.2.0`
 - `prompt-toolkit ==3.0.30`
 - `protobuf ==4.21.2`
 - `psycopg2-binary ==2.9.3`
 - `pyasn1 ==0.4.8`
 - `pyasn1-modules ==0.2.8`
 - `pycparser ==2.21`
 - `pydot ==1.4.2`
 - `PyJWT ==2.4.0`
 - `pykwalify ==1.8.0`
 - `pymongo ==4.1.1`
 - `pyparsing ==3.0.9`
 - `pyrsistent ==0.18.1`
 - `pyTelegramBotAPI ==4.6.0`
 - `python-crfsuite ==0.9.8`
 - `python-dateutil ==2.8.2`
 - `python-engineio ==4.3.3`
 - `python-socketio ==5.7.1`
 - `pytz ==2022.1`
 - `PyYAML ==6.0`
 - `questionary ==1.10.0`
 - `randomname ==0.1.5`
 - `rasa ==3.2.2`
 - `rasa-sdk ==3.2.0`
 - `redis ==4.3.4`
 - `regex ==2022.7.9`
 - `requests ==2.28.1`
 - `requests-oauthlib ==1.3.1`
 - `requests-toolbelt ==0.9.1`
 - `rocketchat-API ==1.25.0`
 - `rsa ==4.8`
 - `ruamel.yaml ==0.17.21`
 - `ruamel.yaml.clib ==0.2.6`
 - `s3transfer ==0.6.0`
 - `sanic ==22.6.0`
 - `Sanic-Cors ==2.0.1`
 - `sanic-jwt ==1.8.0`
 - `sanic-routing ==22.3.0`
 - `scikit-learn ==1.1.1`
 - `scipy ==1.8.1`
 - `sentry-sdk ==1.7.2`
 - `six ==1.16.0`
 - `sklearn-crfsuite ==0.3.6`
 - `slackclient ==2.9.4`
 - `SQLAlchemy ==1.4.39`
 - `tabulate ==0.8.10`
 - `tarsafe ==0.0.3`
 - `tensorboard ==2.9.1`
 - `tensorboard-data-server ==0.6.1`
 - `tensorboard-plugin-wit ==1.8.1`
 - `tensorflow ==2.9.1`
 - `tensorflow-addons ==0.17.1`
 - `tensorflow-estimator ==2.9.0`
 - `tensorflow-hub ==0.12.0`
 - `tensorflow-probability ==0.17.0`
 - `tensorflow-text ==2.9.0`
 - `termcolor ==1.1.0`
 - `terminaltables ==3.1.10`
 - `threadpoolctl ==3.1.0`
 - `toolz ==0.12.0`
 - `tqdm ==4.64.0`
 - `twilio ==7.11.0`
 - `typeguard ==2.13.3`
 - `typing-extensions ==4.3.0`
 - `typing-utils ==0.1.0`
 - `tzlocal ==4.2`
 - `ujson ==5.4.0`
 - `urllib3 ==1.26.10`
 - `uvloop ==0.16.0`
 - `wcwidth ==0.2.5`
 - `webexteamssdk ==1.6.1`
 - `websockets ==10.3`
 - `Werkzeug ==2.1.2`
 - `wrapt ==1.14.1`
 - `yarl ==1.7.2`
 - `zipp ==3.8.1`

</details>

</blockquote>
</details>
